spec/spec_models.cr

```diff
@@ -352,7 +352,14 @@ require "uuid"
     adapter {{ adapter_literal }}
     table_name uuids
 
-    primary uuid : String, auto: :uuid
+    primary uuid : UUID, converter: Granite::Converters::UuidConverter
+  end
+
+  class UUIDNaturalModel < Granite::Base
+    adapter {{ adapter_literal }}
+    table_name uuids
+
+    primary uuid : UUID, converter: Granite::Converters::UuidConverter, auto: false
```

Comment:
Can you define a `field uuid : UUID` as well?

Reply:
Yea, should be able to.  I can add a spec for `field` and `field!` as well.  Would of course have to set the converter on them tho.